schemas/com.snowplowanalytics.snowplow.enrichments/pii_enrichment_config/jsonschema/1-1-0

```diff
@@ -0,0 +1,137 @@
+                      "user_id",
+                      "user_ipaddress",
+                      "user_fingerprint",
+                      "domain_userid",
+                      "network_userid",
+                      "ip_organization",
+                      "ip_domain",
+                      "tr_orderid",
+                      "ti_orderid",
+                      "mkt_term",
+                      "mkt_content",
+                      "se_category",
+                      "se_action",
+                      "se_label",
+                      "se_property",
+                      "mkt_clickid",
+                      "refr_domain_userid",
+                      "domain_sessionid"
+                    ]
+                  }
+                },
+                "required": [
+                  "field"
+                ],
+                "additionalProperties": false
+              },
+              "json": {
+                "type": "object",
+                "properties": {
+                  "field": {
+                    "type": "string"
+                  },
+                  "schemaCriterion": {
+                    "type": "string",
+                    "pattern": "^iglu:([a-zA-Z0-9-_.]+)/([a-zA-Z0-9-_]+)/([a-zA-Z0-9-_]+)/([1-9][0-9]*|\\*)-((?:0|[1-9][0-9]*)|\\*)-((?:0|[1-9][0-9]*)|\\*)$"
+                  },
+                  "jsonPath": {
+                    "type": "string",
+                    "pattern": "^\\$.*$"
+                  }
+                },
+                "required": [
+                  "field",
+                  "schemaCriterion",
+                  "jsonPath"
+                ],
+                "additionalProperties": false
+              }
+            },
+            "oneOf": [
+              {
+                "required": [
+                  "pojo"
+                ]
+              },
+              {
+                "required": [
+                  "json"
+                ]
+              }
+            ],
+            "additionalProperties": false
+          }
+        },
+        "strategy": {
+          "type": "object",
+          "properties": {
+            "pseudonymize": {
+              "type": "object",
+              "properties": {
+                "hashFunction": {
+                  "enum": [
+                    "MD2",
+                    "MD5",
+                    "SHA-1",
+                    "SHA-256",
+                    "SHA-384",
+                    "SHA-512"
+                  ]
+                }
+              },
+              "required": [
+                "hashFunction"
+              ],
+              "additionalProperties": false
+            }
+          },
+          "required": [
+            "pseudonymize"
+          ],
+          "additionalProperties": false
+        }
+      },
+      "required": [
+        "pii",
+        "strategy"
+      ],
+      "additionalProperties": false
+    }
+  },
+  "required": [
+    "enabled",
+    "emitEvent",
```

Comment:
Technically this requirement here makes the schema a 2-0-0 (breaking change), because a 1-0-0 instance will fail validation against this schema now. @chuwy do you have a preference on what we do - bump the version or make the field not-required?

Reply:
This is 1-1-0 if not required, as this is not a fix right?

Reply:
No, the middle digit refers to a schema widening (we call it a `REVISION`), not a `MINOR` - SchemaVer is not SemVer. It's my bad for saying it was 1-1-0 - it should be either 2-0-0 or 1-0-1... Let's see what @chuwy says.

Reply:
Am I wrong in assumption that there's exactly 0 instances with this schema attached in the wild. If this assumption is correct then I prefer to patch it actually (`1-0-0` again).

But if above assumption is wrong then in general, for configurations I prefer to **not** avoid MODEL bumps - user has to change `schema` anyway, let's make it clear that there's breaking changes instead of loosening schema.

Reply:
> Am I wrong in assumption that there's exactly 0 instances with this schema attached in the wild. If this assumption is correct then I prefer to patch it actually (1-0-0 again).

This assumption will be wrong by the time this version of the schema is released.

> for configurations I prefer to not avoid MODEL bumps - user has to change schema anyway, let's make it clear that there's breaking changes instead of loosening schema.

Sounds good, 2-0-0 it is!

Reply:
> This assumption will be wrong by the time this version of the schema is released.

Sorry for confusion - I meant `1-0-0`. PII enrichment doesn't exist de-facto, so there's just nothing can be incompatible yet.

Reply:
Not sure I follow. By the time this version is released, 1-0-0 will be out in the wild.

Reply:
I think I'm missing some detail as well... anyway I don't have a strong preference, let it be `2-0-0`.